starter/src/assets/script.js

```diff
@@ -14,60 +12,80 @@
+// Helper function to get product
+// Most recent product interacted with
 
-/* Create a function named addProductToCart that takes in the product productId as an argument
-  - addProductToCart should get the correct product based on the productId
-  - addProductToCart should then increase the product's quantity
+
+// Gets the product from the SKU received from the frontend
+
+
+/* Create a function named addToCart that takes in the product SKU as an argument
+  - addToCart should get the correct product based on the SKU
+  - addToCart should then increase the product's quantity
   - if the product is not already in the cart, add it to the cart
 */
 
-/* Create a function named increaseQuantity that takes in the productId as an argument
-  - increaseQuantity should get the correct product based on the productId
-  - increaseQuantity should then increase the product's quantity
+
+/* Create a function named increase that takes in the product SKU as an argument
+  - increase should get the correct product based on the SKU
+  - increase should then increase the product's quantity
 */
 
-/* Create a function named decreaseQuantity that takes in the productId as an argument
-  - decreaseQuantity should get the correct product based on the productId
-  - decreaseQuantity should decrease the quantity of the product
+
+/* Create a function named decrease that takes in the product SKU as an argument
+  - decrease should get the correct product based on the SKU
+  - decrease should decrease the quantity of the product
   - if the function decreases the quantity to 0, the product is removed from the cart
 */
 
-/* Create a function named removeProductFromCart that takes in the productId as an argument
-  - removeProductFromCart should get the correct product based on the productId
-  - removeProductFromCart should update the product quantity to 0
-  - removeProductFromCart should remove the product from the cart
+
+/* Create a function named remove that takes in the product SKU as an argument
+  - remove should get the correct product based on the SKU
+  - remove should update the product quantity to 0
+  - remove should remove the product from the cart
 */
 
+
+/* When paying, the user should be able to split their payment */
+/* Create a global variable named totalPaid to store the current remaining balance */
+
+
 /* Create a function named cartTotal that has no parameters
   - cartTotal should iterate through the cart to get the total of all products
   - cartTotal should return the sum of the products in the cart
 */
 
-/* Create a function called emptyCart that empties the products from the cart */
 
 /* Create a function named pay that takes in an amount as an argument
   - pay will return a negative number if there is a remaining balance
   - pay will return a positive number if money should be returned to customer
 */
 
-/* Place stand out suggestions here (stand out suggestions can be found at the bottom of the project rubric.)*/
```

Comment:
Not sure why this hint is removed.

Reply:
For this one I was just hoping to add in parenthesis where the standout suggestions can be found, for example - (stand out suggestions can be found at the bottom of the project rubric.)

Reply:
Ok.  We can do that